src/Interfaces/MarketingSuite/Assets/HasAssetDetails.php

```diff
@@ -0,0 +1,18 @@
+<?php declare(strict_types = 1);
+
+namespace Vicimus\Support\Interfaces\MarketingSuite\Assets;
+
+/**
+ * Interface HasAssetDetails
+ */
+interface HasAssetDetails
+{
+    /**
+     * Asset errored with the provided payload
+     *
+     * @param string $message The error message
+     *
+     * @return void
+     */
+    public function error(string $message): void;
```

Comment:
Do you think this should be setError or do you think it's clear enough it's functionality.